src/ucp/api/ucp.h

```diff
@@ -1607,6 +1646,19 @@ struct ucp_am_recv_param {
 };
 
 
+/**
+ * @ingroup UCP_CONTEXT
+ * @brief Get attributes of the UCP library.
+ *
+ * This routine fetches information about the library.
```

Comment:
"about the library" -> "about the library attributes"  or "library configuration"

Reply:
the "attributes" version sounds good